experiments/simbricks/orchestration/runtime/distributed.py

```diff
@@ -71,6 +71,8 @@ async def do_run(self, run: Run):
         run.output = await runner.run()  # already handles CancelledError
         self.complete.append(run)
 
+        # if the log is huge, this step takes some time
+        print(f'Writing collected output of run {run.name()} to JSON file ...')
```

Comment:
I would suggest only enabling the print if --verbose is set, as without it it's currently completely quiet.